### ISSUE TITLE
Unhandled match case NULL

### ISSUE THREAD

**Issue by deibertf** [ploi/roadmap#204]
Just upgraded to the latest version.
Settings page throws an error:
`app/Filament/Pages/Settings.php: 219` -> `Unhandled match case NULL`

- PHP 8.1.6
- Laravel 9.52.0

```php
Select::make('projects')
    ->multiple()
    ->helperText('Optionally select projects to trigger for, if you do not select a project it will always notify on new events')
    ->options(Project::pluck('title', 'id')),
    TextInput::make('name')->label(function ($get) {
        return match ($get('type')) {
            'email' => 'Name receiver',
            'discord', 'slack' => 'Label'
    };
})->required()
```

**Comment by Cannonb4ll** [comment]
Interesting, unable to replicate it. Did you run migrations?

**Comment by deibertf** [comment]
It seems, actually its not the latest release but the main branch :/ sry for the confusion.
Site ID is https://ploi.io/panel/servers/23232/sites/103352 if that may help you out

**Comment by Cannonb4ll** [comment]
Can you show me how your settings table looks like in your database?

**Comment by deibertf** [comment]
![image](https://user-images.githubusercontent.com/24978665/220617767-f84649c1-500e-4dba-b488-e3dbf3ad39cd.png)

**Comment by Cannonb4ll** [comment]
I meant, the contents of all rows currently, or an export of your settings table will help as well. The structure is not important.

**Comment by deibertf** [comment]
> I meant, the contents of all rows currently, or an export of your settings table will help as well. The structure is not important.

sent you and export via discord

**Comment by Cannonb4ll** [comment]
Should be fixed in #d336a601962989027916aac32b1cd4647a80beb8
